.github/workflows/ruby.yml

```diff
@@ -0,0 +1,50 @@
+name: Ruby Gem
+
+on: [push]
+
+jobs:
+  build:
+    name: Build
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        ruby: [2.3.x, 2.4.x, 2.5.x, 2.6.x]
+
+    steps:
+      - uses: actions/checkout@master
```

Comment:
Is there a reason that that you're using the master version of checkout instead of v1. 

https://help.github.com/en/articles/configuring-a-workflow#using-the-checkout-action